applications/ShallowWaterApplication/custom_utilities/interpolate_sw_to_pfem_utility.hpp

```diff
@@ -64,54 +63,56 @@ class InterpolateSwToPfemUtility
     void InterpolateVariables (ModelPart& rPfemModelPart, ModelPart& rSWModelPart)
     {
         
-        const int NNodesSW = static_cast<int>(rSWModelPart.Nodes().size());
+        long unsigned int NNodesSW = static_cast<int>(rSWModelPart.Nodes().size());
```

Comment:
in Kratos is uncommon to use this type. we would use `const std::size_t nnodes_sw = rSWModelPart.Nodes().size()`